ansible_collections/arista/avd/plugins/plugin_utils/utils/anta_logging_filter.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2025 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+import logging
+
+
+class AntaLoggingFilter(logging.Filter):
+    """Filter logs for ANTA and its underlying libraries."""
+
+    _anta_libraries = ("anta", "asyncio", "asyncssh", "httpcore", "httpx")
+
+    def __init__(self, has_warnings_ref: list[bool] | None = None, *, exclude: bool = True) -> None:
+        """Initialize the filter.
+
+        Parameters
+        ----------
```

Comment:
in pyavd / AVD we follow google style docstring (we may want to align our libs)
```suggestion
        Args:
```

Reply:
Updated